orders/migrations/0003_static_orders.py

```diff
@@ -0,0 +1,79 @@
+              timezone.datetime(2022, 6, 7), timezone.datetime(2022, 7, 15), timezone.datetime(2021, 1, 20),
+              timezone.datetime(2023, 6, 6), timezone.datetime(2021, 5, 6), timezone.datetime(2023, 4, 20),
+              timezone.datetime(2022, 3, 30), timezone.datetime(2022, 7, 25), timezone.datetime(2022, 10, 10),
+              timezone.datetime(2022, 12, 15), timezone.datetime(2022, 5, 11), timezone.datetime(2022, 9, 10),
+              timezone.datetime(2022, 6, 11), timezone.datetime(2022, 5, 5), timezone.datetime(2022, 2, 22),
+              timezone.datetime(2022, 7, 8), timezone.datetime(2022, 8, 1), timezone.datetime(2022, 1, 1),
+              timezone.datetime(2022, 7, 7), timezone.datetime(2022, 7, 20), timezone.datetime(2022, 8, 27),
+              timezone.datetime(2022, 1, 15), timezone.datetime(2022, 12, 18), timezone.datetime(2022, 9, 23),
+              timezone.datetime(2022, 11, 12), timezone.datetime(2022, 11, 11), timezone.datetime(2022, 1, 23),
+              timezone.datetime(2022, 6, 12), timezone.datetime(2022, 2, 15), timezone.datetime(2022, 3, 11),
+              timezone.datetime(2022, 1, 17), timezone.datetime(2022, 12, 23), timezone.datetime(2022, 3, 12),
+              timezone.datetime(2022, 3, 5), timezone.datetime(2022, 4, 4), timezone.datetime(2022, 5, 15),
+              timezone.datetime(2022, 5, 5)]
+
+END_DATE = [timezone.datetime(2022, 3, 30), timezone.datetime(2022, 5, 25), timezone.datetime(2022, 3, 1),
+            timezone.datetime(2022, 6, 17), timezone.datetime(2022, 7, 25), timezone.datetime(2021, 1, 23),
+            timezone.datetime(2023, 6, 13), timezone.datetime(2021, 5, 26), timezone.datetime(2023, 4, 28),
+            timezone.datetime(2022, 4, 5), timezone.datetime(2022, 7, 29), timezone.datetime(2022, 10, 16),
+            timezone.datetime(2022, 12, 20), timezone.datetime(2022, 5, 14), timezone.datetime(2022, 9, 18),
+            timezone.datetime(2022, 6, 20), timezone.datetime(2022, 5, 24), timezone.datetime(2022, 2, 28),
+            timezone.datetime(2022, 7, 14), timezone.datetime(2022, 8, 19), timezone.datetime(2022, 1, 17),
+            timezone.datetime(2022, 7, 17), timezone.datetime(2022, 7, 25), timezone.datetime(2022, 9, 5),
+            timezone.datetime(2022, 1, 24), timezone.datetime(2022, 12, 24), timezone.datetime(2022, 10, 1),
+            timezone.datetime(2022, 11, 24), timezone.datetime(2022, 11, 22), timezone.datetime(2022, 2, 3),
+            timezone.datetime(2022, 6, 20), timezone.datetime(2022, 2, 28), timezone.datetime(2022, 3, 25),
+            timezone.datetime(2022, 1, 22), timezone.datetime(2022, 12, 26), timezone.datetime(2022, 3, 26),
+            timezone.datetime(2022, 3, 17), timezone.datetime(2022, 4, 17), timezone.datetime(2022, 5, 26),
+            timezone.datetime(2022, 5, 14)]
+
+PRICE_PER_DAY = [200, 230, 150, 60, 160, 300, 90, 150, 170, 50, 200, 230, 150, 60, 160, 300, 90, 150, 170, 50, 200, 230,
```

Comment:
why not use the daycare field price per day?

Reply:
You're right. Good one.